File: Java/topflytechCodec/src/com/topflytech/codec/entities/Rs232FuelMessage.java
package com.topflytech.codec.entities;

/**
 * Created by admin on 2017/11/27.
 */
public class Rs232FuelMessage extends Rs232DeviceMessage {


    public void setFuelPercent(Float fuelPercent) {
        this.fuelPercent = fuelPercent;
    }

    public Float getTemp() {
        return temp;
    }

    public void setTemp(Float temp) {
        this.temp = temp;
    }

    public Integer getLiquidType() {
        return liquidType;
    }

    public void setLiquidType(Integer liquidType) {
        this.liquidType = liquidType;
    }

    public Float getCurLiquidHeight() {
        return curLiquidHeight;
    }

    public void setCurLiquidHeight(Float curLiquidHeight) {
        this.curLiquidHeight = curLiquidHeight;
    }

    public Float getFullLiquidHeight() {
        return fullLiquidHeight;
    }

    public void setFullLiquidHeight(Float fullLiquidHeight) {
        this.fullLiquidHeight = fullLiquidHeight;
    }

    public Float getFuelPercent() {
        return fuelPercent;
    }
    public int getAlarm() {
        return alarm;
    }

    public void setAlarm(int alarm) {
        this.alarm = alarm;
    }
    private Float fuelPercent;
    private Float temp;
    private Integer liquidType;
    private Float curLiquidHeight;
    private Float fullLiquidHeight;



    private int alarm = 0;
    public final static int LIQUID_TYPE_OF_DIESEL = 1;
    public final static int LIQUID_TYPE_OF_PETROL = 2;
    public final static int LIQUID_TYPE_OF_WATER = 3;

    public final static int ALARM_NONE = 0;
    public final static int ALARM_FILL_TANK = 1;
    public final static int ALARM_FUEL_LEAK = 2;
}

File: C#/TopflytechCodec/TopflytechCodec/Entities/Rs232FuelMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    class Rs232FuelMessage : Rs232DeviceMessage
    {
        private float fuelPercent;

        public float FuelPercent
        {
            get { return fuelPercent; }
            set { fuelPercent = value; }
        }
        private float temp;

        public float Temp
        {
            get { return temp; }
            set { temp = value; }
        }
        private int liquidType;

        public int LiquidType
        {
            get { return liquidType; }
            set { liquidType = value; }
        }
        private float curLiquidHeight;

        public float CurLiquidHeight
        {
            get { return curLiquidHeight; }
            set { curLiquidHeight = value; }
        }
        private float fullLiquidHeight;

        public float FullLiquidHeight
        {
            get { return fullLiquidHeight; }
            set { fullLiquidHeight = value; }
        }
        private int alarm = 0;

        public int Alarm
        {
            get { return alarm; }
            set { alarm = value; }
        }


       
        public static int LIQUID_TYPE_OF_DIESEL = 1;
        public static int LIQUID_TYPE_OF_PETROL = 2;
        public static int LIQUID_TYPE_OF_WATER = 3;

        public static int ALARM_NONE = 0;
        public static int ALARM_FILL_TANK = 1;
        public static int ALARM_FUEL_LEAK = 2;
    }
}
